webboot/webboot.go

```diff
@@ -148,15 +132,25 @@ func main() {
 
 	//Processes the URL to receive an io.ReadCloser, which holds the content of the downloaded file
 	log.Println("Retrieving the file...")
-	read, err := linkOpen(URL)
+	iso, err := linkOpen(URL)
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	err = write(read, filename)
+	// TODO: Find a persistent memory device large enough to store the content. If no blocks are available, error the user.
+	pmem, err := os.OpenFile("/dev/pmem0", os.O_APPEND|os.O_WRONLY, 0600)
 	if err != nil {
 		log.Fatal(err)
 	}
+	if _, err := io.Copy(pmem, iso); err != nil {
+		log.Fatalf("Error copying to persistent memory device: %v", err)
+	}
+	if err = iso.Close(); err != nil {
```

Comment:
I have a question here: what is a good practice for defering Close() and checking the error for Close() right after I am done utilizing it? I figured since I am done using the io.Reader here, that I can just check the error if I receive an error from closing here? However, I can also defer a Close() once main() is done executing..